tests/sentry/tasks/integrations/github/test_open_pr_comment.py

```diff
@@ -399,8 +399,10 @@ def test_not_first_frame(self):
 
         top_5_issues = get_top_5_issues_by_count_for_file([self.project], ["baz.py"], ["world"])
         top_5_issue_ids = [issue["group_id"] for issue in top_5_issues]
+        function_names = [issue["function_name"] for issue in top_5_issues]
         assert group_id != self.group_id
         assert top_5_issue_ids == [self.group_id, group_id]
+        assert function_names == ["world", "world"]
```

Comment:
Why does this return two filenames?

Reply:
it returns 2 issues, both of which match the single function name `world` that is passed into the query. one issue has the function name at the top of the stacktrace and the other has it at the second frame. this test fails without the change to the snuba query

Reply:
also it's a function name

Reply:
I see, thanks for the clarification